app/src/main/res/values/localpages.xml

```diff
@@ -7,207 +7,6 @@
     <string name="errorpage_title">Problem loading page</string>
```

Comment:
these are also defined in the A-C strings. can we use those instead?

Reply:
Good catch. Thanks!